ui/app/src/Utils/Api.elm

```diff
@@ -66,7 +66,7 @@ iso8601Time =
 
 baseUrl : String
 baseUrl =
-    "/api/v1"
+    "http://localhost:9093/api/v1"
```

Comment:
@mxinden right, we would need to serve this page instead of including it as a static asset